http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java

```diff
@@ -717,7 +717,7 @@ public void testCompressionIsDisabled()
         assertEquals(body, "");
         assertFalse(servlet.getRequestHeaders().containsKey(HeaderName.of(ACCEPT_ENCODING)));
 
-        String json = "{\"foo\":\"bar\",\"hello\":\"world\"}";
+        String json = "{\"fuite\":\"apple\",\"hello\":\"world\"}";
```

Comment:
Unrelated change?

Reply:
No, the minimum size for gzip increased.